inference/server/oasst_inference_server/settings.py

```diff
@@ -10,6 +10,7 @@ def split_keys_string(keys: str | None):
 
 
 class Settings(pydantic.BaseSettings):
+    PROJECT_NAME: str = "open-assistant inference server"
```

Comment:
all other settings are lower-snake-case .. maybe we should be consistent here and make it `project_name`?

Reply:
Yeah, was just copying the equivalent in the backend code as close as possible that's only reason I did this way. No strong feelings either way.